services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.repository.ts

```diff
@@ -20,7 +20,7 @@ export class ProgramFinancialServiceProviderConfigurationRepository extends Repo
     );
   }
 
-  public async findByProgramIdAndFinancialServiceProviderName({
+  public async getByProgramIdAndFinancialServiceProviderName({
     programId,
     financialServiceProviderName,
     includeProperties,
```

Comment:
I suggest to remove this parameter and let the function always return the properties. A bit of overhead because data may be retrieved that is not used, but making it simpler for the caller not to be bothered with this IMO outweighs that. At the moment this function is only called once with true.